### PR TITLE
Vmware fusion support

### PR DESCRIPTION
**Pull request by joshfng**
Simple vmware fusion support. I'll address specific changes as line comments.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -2,17 +2,20 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 	# Configure The Box
-	config.vm.box = "ubuntu/trusty64"
+	config.vm.box = "chef/ubuntu-14.10"
```
Comment (joshfng) [inline Vagrantfile]: Switching to a community box with proper vmware support.

File: Vagrantfile [hunk 2]
```diff
@@ -2,17 +2,20 @@ VAGRANTFILE_API_VERSION = "2"
...
 	config.vm.hostname = "homestead"
 
-	config.vm.network :private_network, ip: "192.168.33.10"
```
Comment (joshfng) [inline Vagrantfile]: No need for a private network on this box.

File: scripts/provision.sh [hunk 3]
```diff
@@ -174,11 +174,9 @@ groups vagrant
 # Install Node
 
 apt-get install -y nodejs
-sudo su vagrant <<'EOF'
```
Comment (joshfng) [inline scripts/provision.sh]: This command was failing as the vagrant user. Fixed it so it works and installs the npm packages globally as expected

File: scripts/provision.sh [hunk 4]
```diff
@@ -192,12 +190,12 @@ apt-get install -y mysql-server-5.6
 
 # Configure MySQL Remote Access
 
-sed -i '/^bind-address/s/bind-address.*=.*/bind-address = 10.0.2.15/' /etc/mysql/my.cnf
-mysql --user="root" --password="secret" -e "GRANT ALL ON *.* TO root@'10.0.2.2' IDENTIFIED BY 'secret' WITH GRANT OPTION;"
+sed -i '/^bind-address/s/bind-address.*=.*/bind-address = 0.0.0.0/' /etc/mysql/my.cnf
```
Comment (joshfng) [inline scripts/provision.sh]: Changed mysql to bind to all available interfaces within the vm, as the internal ip addresses are not the same between vmware and virtualbox.

File: scripts/provision.sh [hunk 5]
```diff
@@ -237,7 +235,3 @@ sudo /etc/init.d/beanstalkd start
 /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=1024
 /sbin/mkswap /var/swap.1
 /sbin/swapon /var/swap.1
-
-# Write Bash Aliases
-
-cp /vagrant/aliases /home/vagrant/.bash_aliases
```
Comment (joshfng) [inline scripts/provision.sh]: There is no alias directory in this repo to mount into the box. It was causing provisioning to fail with directory not found.

**Comment by joshfng** [comment]
Once these are built and uploaded to Atlas, all a Homestead user would have to is `export VAGRANT_DEFAULT_PROVIDER=vmware_fusion` in their shell/.bash file to be able to `homestead up` with vmware.

**Comment by taylorotwell** [comment]
So I guess you have tested this build with both providers?

**Comment by joshfng** [comment]
Yes I've built and tested both virtualbox and vmware. Everything is working as intended.

**Comment by taylorotwell** [comment]
Cool, will pull it down and test it out.

**Comment by taylorotwell** [comment]
Done

**Comment by joshfng** [comment]
No merge love?

**Comment by taylorotwell** [comment]
Read the commit log

**Comment by taylorotwell** [comment]
They are in there

**Comment by joshfng** [comment]
Ah I see them, sorry about that. Thanks!

**Comment by gdhnz** [comment]
The box seems to be missing the blackfire-agent and blackfire-php packages.  I had to install them manually to get blackfire to work.  Apart from that.  Thanks for all your work.

**Comment by taylorotwell** [comment]
@joshfng i've also had reports of the box missing HHVM.

**Comment by taylorotwell** [comment]
Which we will need to fix both of these issues. Can you look into them?

**Comment by gdhnz** [comment]
It also appears after.sh is not running either.

**Comment by joshfng** [comment]
Very odd, it's running the same provision script the virtualbox run. I'll check it out.

**Comment by joshfng** [comment]
Fixed in https://github.com/laravel/settler/pull/36

**Comment by gdhnz** [comment]
Trashing my .homestead and re-reunning homestead init, adding my after.sh, and it now seems to be used when I provision homestead.

**Comment by joshfng** [comment]
@gdhnz can you `homestead update` to 0.2.5 and verify your blackfire config is working please?

**Comment by gdhnz** [comment]
Blackfire working as expected in 0.2.5.  Thanks.

**Pull request closed without merging**